data/formations_basic.txt

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2019-2021 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+# Formation where ships form a line behind the formation leader.
+formation "Trailing line"
+	flippable y
+	position 0 200
+	position 0 400
+	position 0 600
+	position 0 800
+	position 0 1000
+	position 0 1200
+	position 0 1400
+	position 0 1800
```

Comment:
```suggestion
	position 0 1400
	position 0 1600
	position 0 1800
```

Did you intentionally leave a space in the pattern?  I don't know if I prefer a space or not.

Reply:
It was not intentional. Thanks for noticing.

Reply:
Having a space might be an interesting idea to separate certain groups. As an example freighters and combat escorts in 2 different groups. I sort of had that happen with this unintentional gap. I ended up with one group (Type A) then a gap and the second group (Type B). I'm not sure how the order is sorted but I assume it was to do with my naming scheme. 
I use something like this:

- Type A l
- Type A ll
- Type A lll
- Type A lV
Etc. Etc.
- Type B l
- Type B ll
- Type B lll
Etc. Etc.